ts/features/idpay/unsubscription/screens/UnsubscriptionConfirmationScreen.tsx

```diff
@@ -0,0 +1,165 @@
+    }
+  ];
+
+const UnsubscriptionConfirmationScreen = () => {
+  const machine = useUnsubscriptionMachineService();
+
+  const isLoading = useSelector(machine, isLoadingSelector);
+  const initiativeName = useSelector(machine, selectInitiativeName);
+
+  const checks = useConfirmationChecks(unsubscriptionChecks.length);
+
+  const handleClosePress = () =>
+    machine.send({
+      type: "EXIT"
+    });
+
+  const handleConfirmPress = () => {
+    machine.send({
+      type: "CONFIRM_UNSUBSCRIPTION"
+    });
+  };
+
+  const handleCheckToggle = (index: number) => checks.toggle(index);
+
+  const closeButton = (
+    <TouchableDefaultOpacity
+      onPress={handleClosePress}
+      accessible={true}
+      accessibilityLabel={I18n.t("global.buttons.back")}
+      accessibilityRole={"button"}
+    >
+      <IconFont name={"io-close"} style={{ color: IOColors.bluegrey }} />
+    </TouchableDefaultOpacity>
+  );
+
+  const confirmModal = useIOBottomSheetModal(
+    <Body>{I18n.t("idpay.unsubscription.modal.content")}</Body>,
+
+    I18n.t("idpay.unsubscription.modal.title", { initiativeName }),
+    250,
+
+    <FooterWithButtons
+      type="TwoButtonsInlineHalf"
+      leftButton={{
+        onPress: () => {
+          confirmModal.dismiss();
+          handleConfirmPress();
+        },
+        block: true,
+        bordered: true,
+        title: I18n.t("idpay.unsubscription.button.continue"),
+        danger: true
+      }}
+      rightButton={{
+        onPress: () => {
+          confirmModal.dismiss();
+        },
+        block: true,
+        bordered: true,
+        title: I18n.t("global.buttons.cancel"),
+        labelColor: IOColors.blue
+      }}
+    />
+  );
+
+  const body = (
+    <SafeAreaView style={IOStyles.flex}>
+      <View style={styles.content}>
+        <H1>{I18n.t("idpay.unsubscription.title", { initiativeName })}</H1>
+        <VSpacer size={16} />
+        <Body>{I18n.t("idpay.unsubscription.subtitle")}</Body>
+        <VSpacer size={16} />
+        <FlatList
+          data={unsubscriptionChecks}
+          renderItem={({ item, index }) => (
+            <UnsubscriptionCheckListItem
+              key={index}
+              title={item.title}
+              subtitle={item.subtitle}
+              checked={checks.values[index]}
+              onValueChange={() => handleCheckToggle(index)}
+            />
+          )}
+        />
+      </View>
+      <FooterWithButtons
+        type={"TwoButtonsInlineHalf"}
+        leftButton={{
+          bordered: true,
+          title: I18n.t("global.buttons.cancel"),
+          onPress: handleClosePress
+        }}
+        rightButton={{
+          title: I18n.t("idpay.unsubscription.button.continue"),
+          onPress: confirmModal.present,
+          disabled: !checks.areFulfilled,
+          danger: checks.areFulfilled
+        }}
+      />
+    </SafeAreaView>
+  );
+
+  return (
+    <BaseScreenComponent
+      goBack={true}
+      headerTitle="Rimuovi iniziativa"
```

Comment:
Might it make sense to use a locale for header title?

Reply:
You are right! Fixed in https://github.com/pagopa/io-app/pull/4462/commits/c57406a18a276ede84b7996f61f56e0784c9bfa7